### ISSUE TITLE
[🐞]: mouseClicked event handler

### ISSUE THREAD

**Issue by 0Starocean0** [MehVahdJukaar/Moonlight#285]
### BEFORE CONTINUING:

- [X] Always check you are using the latest version of these mods and its dependencies
- [X] Remove known problematic rendering mods such as Optifine, Sodium and others

### Version-Loader

1.21.1-FABRIC

### Moonlight Lib Version

1.21-2.17.12

### Describe the Issue

Crashed when creating new world

### Log Attachment

https://mclo.gs/2Z7CzQx

### OPTIONAL: Which mods are affected?

Supplementaries

**Comment by MehVahdJukaar** [comment]
Try without other mods on with just supplementarirs

**Comment by 0Starocean0** [comment]
Didn't find which causes this, it's familiar to issue#277

**Comment by MehVahdJukaar** [comment]
meaning it happens or does not happens without other mods?

**Comment by 0Starocean0** [comment]
It only happens with cyanide mod

**Comment by MehVahdJukaar** [comment]
Yup. I know of this issue then. That mod is nuking other mod added Datapack registries. Are you sure you have it updated because about a year ago this was reported and I was sure they sai they fixed it. If not close this and open an issue on there again
